.github/workflows/release.yml

```
@@ -12,5 +12,6 @@ jobs:
     uses: fingerprintjs/dx-team-toolkit/.github/workflows/release-typescript-project.yml@v1
     with:
       nodeVersion: 16
```

```suggestion
      nodeVersion: 20
```
Let's also bump the node version here.